### PR TITLE
Custom cast range ability

### PR DESCRIPTION
**Pull request by Yahnich**
working

### CODE REVIEW

File: src/scripts/npc/npc_abilities_custom.txt [hunk 1]
```diff
@@ -2997,5 +2997,39 @@
...
+        "AbilityTextureName"            "custom_cast_range"
 
+        // Special
+        //-------------------------------------------------------------------------------------------------------------
+        "AbilitySpecial"
+        {
+            "01"
+            {
+                "var_type"              "FIELD_INTEGER"
+                "cast_range_bonus"          "100 2000 3000 4000"
```
Comment (darklordabc) [inline src/scripts/npc/npc_abilities_custom.txt]: A little op? jks. You put an extra 0 on some of these.

**Comment by darklordabc** [comment]
works great

**Comment by Myrl** [comment]
Please use 4 spaces rather than hard tabs.

**Comment by Myrl** [comment]
Have you tried using `MODIFIER_PROPERTY_CAST_RANGE_BONUS`? It'd be better if this works with Aether Lens.

**Comment by Yahnich** [comment]
done

**Comment by Yahnich** [comment]
also modifer_cast_range_bonus does not work yet, it's not valve implemented yet

**Comment by Myrl** [comment]
Ah, understood. I'll be testing this right now and will be pulling it if it works properly.

**Comment by Yahnich** [comment]
LGTM

**Pull request closed without merging**